synapse/replication/tcp/client.py

```diff
@@ -221,7 +220,8 @@ async def wait_for_stream_position(
 
         # We insert into the list using heapq as it is more efficient than
         # pushing then resorting each time.
```

Comment:
Seems like this comment should be updated now? 🤨

Reply:
woops